src/components/new-submission/cz.new-submission.vue

```diff
@@ -160,7 +163,7 @@ const renderers = [
   name: "cz-new-submission",
   components: { JsonForms, JsonViewer, CzFolderStructure, CzNewSubmissionActions },
 })
-export default class CzNewSubmission extends mixins<ActiveRepositoryMixin>(ActiveRepositoryMixin) {
+export default class CzNewSubmission extends mixins<ActiveRepositoryMixin>(ActiveRepositoryMixin, IdentifiersMixin) {
```

Comment:
@Maurier I dont know Typescript (I know, I need to learn).
I know I haven't done this right, but not sure how